### ISSUE TITLE
Add missing upper case letters to Italiano Messageease

### ISSUE THREAD

**Issue by ariadne90** [dessalines/thumb-key#635]
- [x] Did you check to see if this issue already exists?
- [x] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [ ] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/thumbkey

**Describe the feature request below**
Hello, I'm not a developer but I love using this app so I created a Github account just for this, I hope I'm following all the steps correctly.

The issue is the following: I settled on using the Italiano Messageease layout because it's the most convenient for me. The problem is that at the time of typing this there are no upper case Y and V letters.

Here's a picture for reference, they should be added on the top right of the T button and the bottom right of the A button, shifting the other letters to match their lower case counterparts.

![thumb-key](https://github.com/dessalines/thumb-key/assets/158187037/2097ae36-5783-430b-95de-64464541e6b9)

**Comment by flauta** [comment]
thanks @ariadne90, I made these errors while creating this keyboard. I use Thumbkey daily but with the IT keyboard. The V and Y didn't appear since they were assigned to the same position of other ones. I submitted the patch now. It will probably appear in the next release.

grazie @ariadne90, ho sbagliato io creando questa disposizione. Uso ThumbKey ma con la tastiera italiana normale per cui non mi ero accorto di aver sovrapposto due lettere. Ho appena inviato la correzione che comparirà probabilmente nella prossima versione.
